File: Exercícios/intervalodeinteiros.c
#include <stdio.h>
#include <locale.h>

int main()
{
    setlocale(LC_ALL, ("pt-br"));

    int numero_digitado, inicio_intervalo, final_intervalo;

    printf("Insira um número que determine o início do intervalo: ");
    scanf("%d", &inicio_intervalo);

    printf("\nInsira um número que determine o final do intervalo: ");
    scanf("%d", &final_intervalo);

    printf("\nInsira um número para checar este intervalo: ");
    scanf("%d", &numero_digitado);

    if (numero_digitado >= inicio_intervalo && numero_digitado <= final_intervalo)
    {
        printf("\nO número %d consta no intervalo!", numero_digitado);
    }
    else
    {
        printf("\nO número %d não consta no intervalo!", numero_digitado);
    }

    return 0;
}

File: Exercícios/intervalodeinteiros.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Insira um número que determine o início do intervalo: ");
        int inicioIntervalo = scanner.nextInt();

        System.out.print("\nInsira um número que determine o final do intervalo: ");
        int finalIntervalo = scanner.nextInt();

        System.out.print("\nInsira um número para checar este intervalo: ");
        int numeroDigitado = scanner.nextInt();

        if (numeroDigitado >= inicioIntervalo && numeroDigitado <= finalIntervalo) {
            System.out.println("\nO número " + numeroDigitado + " consta no intervalo!");
        } else {
            System.out.println("\nO número " + numeroDigitado + " não consta no intervalo!");
        }
    }
}